crontabber/tests/test_crontabber.py

```diff
@@ -2149,6 +2150,105 @@ def test_job_list_separated_sloppily(self):
             ok_('Ran FooJob' in infos, infos)
             ok_('Ran BarJob' in infos, infos)
 
+    def test_run_two_jobs_almost_simultaneously_first_time(self):
```

Comment:
I'm not sure I understand. These two tests do test what the patch is fixing. I wrote the tests first, which failed consistently, then added the actual patch.